### DISCUSSION TITLE
Activation link only as .onion link

### DISCUSSION THREAD

**Discussion by taktgeber** [globaleaks/globaleaks-whistleblowing-software#4236] [Q&A] [answered]
Hi there,

I'am trying to setup a globaleaks instance. At the moment I want to create some test users und test the functionality of the system. After I created a new account, an email with an activation link was sent to the named e-mail address. In this mail the user find an activation link to activation the account and set a password. This link is only available as an .onion link.
Is there a possibility to change the activation link to a normal link?

Thanks in advance, best regards

**Comment by evilaliv3** [comment, marked as answer]
Thank you for your question @taktgeber 

Configure an hostname in the network settings and enable HTTPS and the platform will use that hostname when generating links.
